incubator/solr/values.yaml

```diff
@@ -0,0 +1,97 @@
+---
+
+# Which port should solr listen on
+port: 8983
+
+# Number of solr instances to run
+replicaCount: 3
+
+# Settings for solr java memory
+javaMem: "-Xms2g -Xmx3g"
+
+# Set the limits and requests on solr pod resources
+resources: {}
+
+# Sets the termination Grace period for the solr pods
+# This can take a while for shards to elect new leaders
+terminationGracePeriodSeconds: 180
+
+# Solr image settings
+image:
+  repository: solr
+  tag: 7.6.0
+  pullPolicy: Always
+
+# Solr pod liveness
+livenessProbe:
+  initialDelaySeconds: 20
+  periodSeconds: 10
+
+# Solr pod readiness
+readinessProbe:
+  initialDelaySeconds: 15
+  periodSeconds: 5
+
+# Annotations to apply to the solr pods
+podAnnotations: {}
+
+# Affinity group rules or the solr pods
+affinity: {}
+
+# Update Strategy for solr pods
+updateStrategy:
+  type: "OnDelete"
```

Comment:
Should the update strategy really be `OnDelete`? This requires manual intervention. https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#on-delete

Reply:
I initially set this to `OnDelete` because the solr pods contain data and thought it was better that an administrator was in charge of when the pods rolled during an update. But I now think this is less expected as a default so have changed it.

Reply:
@mattfarina Thank you for taking a look at this and for your expert guidance.